Makefile

```diff
@@ -526,9 +526,12 @@ frontend: node-check $(FOMANTIC_DEST) $(WEBPACK_DEST)
 backend: go-check generate $(EXECUTABLE)
 
 .PHONY: generate
+.NOTPARALLEL: generate
```

Comment:
From what I gather `.NOTPARALLEL` does not take any arguments, just plain if `.NOTPARALLEL:` is anywhere in the makefile, it disables parallelism for the whole makefile.